### PR TITLE
Package Python bindings for our libraries

### PR DESCRIPTION
**Pull request by vpodzime**
For hubs we already package /var/cfengine/lib/python, we should
also do it in non-hub packages.

Ticket: CFE-4281
Changelog: None

Merge together:
https://github.com/cfengine/core/pull/5438
https://github.com/cfengine/buildscripts/pull/1367

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @craigcomstock can review this?

**Review by craigcomstock — APPROVED** [review]

File: packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in [hunk 1]
```diff
@@ -289,17 +289,18 @@ exit 0
...
+
 # PHP modules
 %prefix/lib/php
 #libs needed by apache ldapmodules
 %prefix/lib/apr-util-1
 #libs of postgres
 %{prefix}/lib/postgresql/*
 
-# Python stuff (cfbs)
-%if %{?rhel}%{!?rhel:0} >= 7
```
Comment (olehermanse) [inline packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in:300]: This RHEL 7 if from before could use a coment.

**Review by olehermanse — APPROVED** [review]
LGTM, thanks!

File: packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in (hunk 1 above)
Comment (vpodzime) [reply packaging/cfengine-nova-hub/cfengine-nova-hub.spec.in:300]: True. I guess it was because of no Python 3 on RHEL 6?

**Pull request merged**